diff --git i/drivers/hwmon/nct6683.c w/drivers/hwmon/nct6683.c
index a23047a3b..c199b549a 100644
--- i/drivers/hwmon/nct6683.c
+++ w/drivers/hwmon/nct6683.c
@@ -1202,6 +1202,8 @@ static int nct6683_probe(struct platform_device *pdev)
 	int groups = 0;
 	char build[16];
 
+	pr_info("nct6683_probe\n");
+
 	res = platform_get_resource(pdev, IORESOURCE_IO, 0);
 	if (!devm_request_region(dev, res->start, IOREGION_LENGTH, DRVNAME))
 		return -EBUSY;
@@ -1218,6 +1220,7 @@ static int nct6683_probe(struct platform_device *pdev)
 
 	data->customer_id = nct6683_read16(data, NCT6683_REG_CUSTOMER_ID);
 
+	pr_info("data->customer_id = %x\n", data->customer_id);
 	/* By default only instantiate driver if the customer ID is known */
 	switch (data->customer_id) {
 	case NCT6683_CUSTOMER_ID_INTEL:
@@ -1229,6 +1232,8 @@ static int nct6683_probe(struct platform_device *pdev)
 	case NCT6683_CUSTOMER_ID_ASROCK:
 		break;
 	default:
+	pr_info("customer_id %x unhandled\n", data->customer_id);
+	force = true;
 		if (!force)
 			return -ENODEV;
 	}
@@ -1350,7 +1355,7 @@ static int __init nct6683_find(int sioaddr, struct nct6683_sio_data *sio_data)
 	int addr;
 	u16 val;
 	int err;
-
+	pr_info("nct6683_find\n");
 	err = superio_enter(sioaddr);
 	if (err)
 		return err;
